backend/workflow_manager/workflow_v2/execution.py

```diff
@@ -57,7 +56,6 @@ def __init__(
             tool_instances=tool_instances_as_dto,
             platform_service_api_key=str(platform_key.key),
             ignore_processed_entities=False,
-            include_metadata=include_metadata,
```

Comment:
Is this removed on purpose from v2?

Reply:
@gaya3-zipstack It's removed in V1 by PR [555](https://github.com/Zipstack/unstract/pull/555), So removed here